### PR TITLE
Recipe detail

### PR DESCRIPTION
**Pull request by danhmanh**
image

image

- Display recipe 's detail: name, ingredients, steps, cover photo and photos of each step

image

- Calculate amount of ingredients depends on number of people

image

### CODE REVIEW

**Comment by itssomething** [comment]
ready

**Comment by GitRubyManhPhuc** [comment]
ok

**Comment by danhmanh** [comment]
image
image
đã fix i18n trong js

File: app/models/user.rb [hunk 1]
```diff
@@ -35,8 +35,7 @@ def email_downcase
   end
 
   def avatar_size
-    if avatar.size > Settings.max_file.megabytes
-      errors.add :avatar, t(".over_limit_file_size")
-    end
+    return unless avatar.size > Settings.max_file.megabytes
+    errors.add :avatar, t(".over_limit_file_size")
```
Comment (thanhhungchu95) [inline app/models/user.rb]: trong model phải dùng I18n.t

**Pull request merged**